package.json

```diff
@@ -61,13 +61,15 @@
     "@storybook/addons": "^5.2.1",
     "@storybook/react": "^5.2.1",
     "@storybook/storybook-deployer": "^2.8.1",
-    "autoprefixer": "^9.5.0",
+    "autoprefixer": "^9.7.1",
     "babel-core": "7.0.0-bridge.0",
     "babel-jest": "^24.7.1",
     "babel-loader": "^8.0.5",
     "babel-plugin-transform-react-constant-elements": "^6.23.0",
     "babel-plugin-transform-react-inline-elements": "^6.22.0",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
+    "child_process": "^1.0.2",
```

Comment:
why did these get added? `child_process` and `dotenv`?

Reply:
Sorry, the rest of the notes below are related to me trying to figure out how to publish storybook through github actions. Everything is reverted.